src/store/channels-list/saga.test.ts

```diff
@@ -387,3 +287,12 @@ describe(fetchConversations, () => {
     ]);
   });
 });
+
+function subject(sagaFunction: any, args?: any, additionalProviders = []) {
```

Comment:
I think I'd expect multiple `subject` functions. One per context of which saga we're testing. Not every saga is going to need the same default providers.